libs/defi/shared/src/components/Activities/components/ActivityIcon.tsx

```diff
@@ -0,0 +1,54 @@
+import { CircularProgress, SvgIcon, useTheme } from '@mui/material';
+import {
+  Activity,
+  FaCircleCheckRegular,
+  FaCircleXmarkRegular,
+} from '@origin/shared/icons';
+
+import type { SvgIconProps } from '@mui/material';
+import type { ReactNode } from 'react';
+
+export type ActivityIconStatus =
+  | 'idle'
+  | 'pending'
+  | 'signed'
```

Comment:
I was unsure if there is a necessary differentiation between `pending` and `signed`. Seeing this in here - I guess you agree that it is valid?

Reply:
it's not really used yet, but I'm ok to keep it as we may want to display another message later